themes/classic/templates/catalog/listing/search.tpl

```diff
@@ -3,3 +3,8 @@
  * You can safely remove it if you want it to appear exactly like all other product listing pages
  *}
 {extends file='catalog/listing/product-list.tpl'}
+
+{block name="errorContent"}
```

Comment:
It's weird to have a block here and not capture like other spots :thinking:

Reply:
Because it extends, instead of include a file :/

Reply:
trololo